randovania/games/prime1/logic_database/Tallon Overworld.txt

```diff
@@ -557,6 +557,16 @@ Extra - aabb: [-703.2118, 297.52924, -49.04736, -619.3062, 378.77765, 64.30938]
   * Extra - nonstandard: False
   > Door to Transport Tunnel B
       Trivial
+  > Door to Arbor Chamber
```

Comment:
I think itd be nicer if you'd have some general node for this OoB shenanigans. So that you can do:
door to root tunnel -> room top => oob requirement s
room top -> pickup (missile expansion) => trivial
room top -> door to arbor chamber => trivial

that way you dont have the requirements duplicated. (ideally have a better name than "room top" tho xP)